test/unit-test/policies/IKeyReleasePolicySnpProps.test.ts

```diff
@@ -0,0 +1,249 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Use the CCF polyfill to mock-up all key-value map functionality for unit-test
+import "@microsoft/ccf-app/polyfill.js";
+import { describe, expect, test } from "@jest/globals";
+import { KeyReleasePolicy } from "../../../src/policies/KeyReleasePolicy";
+import { Logger, LogLevel } from "../../../src/utils/Logger";
+import { IKeyReleasePolicySnpProps } from "../../../src";
+import { IAttestationReport } from "../../../src/attestation/ISnpAttestationReport";
+import { IKeyReleasePolicy } from "../../../src/policies/IKeyReleasePolicy";
+
+describe("Test Key Release Policy properties", () => {
+  test("Should get all data successfully", () => {
+    // Arrange
+    const policy: IKeyReleasePolicySnpProps = {};
+
+    // Act
+    policy["x-ms-attestation-type"] = ["sevsnpvm", "none"];
+
+    // Assert
+    expect(policy["x-ms-attestation-type"]).toContain("sevsnpvm");
+    expect(policy["x-ms-attestation-type"]).toContain("none");
+  });
+
+  test("Should validate successfully", () => {
+    // Arrange
+    Logger.setLogLevel(LogLevel.DEBUG);
+    const policy: IKeyReleasePolicy = {
+      type: "",
+      claims: { "x-ms-attestation-type": ["sevsnpvm"] },
+    };
+    const attestationClaims: IAttestationReport = {
+      "x-ms-attestation-type": "sevsnpvm",
+    };
+
+    // Act
+    const validationResult = KeyReleasePolicy.validateKeyReleasePolicy(
+      policy,
+      attestationClaims,
+    );
+
+    // Assert
+    expect(validationResult.success).toBe(true);
+  });
+  test("Should validate successfully Key Release Policy properties with operator gte", () => {
+    // Arrange
+    Logger.setLogLevel(LogLevel.DEBUG);
+    const policy: any = {
+      type: "",
+      claims: { "x-ms-attestation-type": ["sevsnpvm"] },
```

Comment:
For better coverage, can we have a case where a policy filed have multiple values something like `"x-ms-attestation-type": ["sevsnpvm", "some-other-value"]`?

Reply:
that is the case:
policy["x-ms-attestation-type"] = ["sevsnpvm", "none"];

    // Assert
    expect(policy["x-ms-attestation-type"]).toContain("sevsnpvm");
    expect(policy["x-ms-attestation-type"]).toContain("none");

    Did I miss the point?

Reply:
Can we call `KeyReleasePolicy.validateKeyReleasePolicy` I don't see it in the test?